.circleci/config.yml

```diff
@@ -7,73 +7,90 @@ aliases:
       paths:
         - ~/.lein
         - ~/.m2
+    node_cache: &NODE_CACHE
+      key: lein-{{ checksum "package-lock.json" }}
```

Comment:
```suggestion
      key: node-{{ .Environment.CACHE_VERSION }}-{{ checksum "/tmp/.node_version"
          }}-{{ checksum "package-lock.json" }}
```

Reply:
Maybe add `{{ .Environment.CACHE_VERSION }}` to the lein cache while you are at it